src/test/resources/add-to-cart.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Adding a product to the cart
+
+  Scenario: Searching for a PS5 and adding it to the cart
+    Given User is on the home page 'https://allegro.pl/'
+    And has cookies accepted
+    When User searches for a PS5
+    And selects one of the listed products
```

Comment:
This design is pretty error prone. it's not clear how the selection is being made here. I would prefer step: 'selects FIRST of the listed products'